### PR TITLE
Added healthcheck.html to terminus options.

### PR DESCRIPTION
**Pull request by kawikabader**
## Summary

There is a need to support existing apps that are using the `/healthcheck.html` route instead of the `/healthcheck`.

## Changelog

Added the `/healthcheck.html` route to the Terminus `healthChecks` options.

## Test Plan

Tests were updated.

### CODE REVIEW

File: packages/gasket-plugin-https/index.js [hunk 1]
```diff
@@ -37,8 +37,9 @@ async function start(gasket) {
   if (http2) configOpts.http2 = http2;
 
   const serverOpts = await gasket.execWaterfall('createServers', configOpts);
-  const { healthcheck, ...terminusDefaults } = await gasket.execWaterfall('terminus', {
+  const { healthcheck, healthcheckHtmlRoute, ...terminusDefaults } = await gasket.execWaterfall('terminus', {
     healthcheck: '/healthcheck',
+    healthcheckHtmlRoute: '/healthcheck.html',
```
Comment (kinetifex) [inline packages/gasket-plugin-https/index.js:42]: Glad we can allow for multiple routes. To simplify the config signature, and keep it flexible, could we default `healthcheck` to an array of route names, but which supports a single string if set?
```
const routes = Array.isArray(healthcheck) ? healthcheck : [healthcheck]
```

Comment (runruh-godaddy) [reply packages/gasket-plugin-https/index.js:42]: > Glad we can allow for multiple routes. To simplify the config signature, and keep it flexible, could we default `healthcheck` to an array of route names, but which supports a single string if set?
> 
> ```
> const routes = Array.isArray(healthcheck) ? healthcheck : [healthcheck]
> ```

I love this idea as well

File: packages/gasket-plugin-https/index.js [hunk 2]
```diff
@@ -74,12 +75,15 @@ async function start(gasket) {
     onShutdown: one(async function onShutdown() {
       await gasket.exec('onShutdown');
     }),
-    healthChecks: {
-      [healthcheck]: async function healthCheckRequested() {
-        await gasket.exec('healthcheck', HealthCheckError);
+    healthChecks: (function () {
```
Comment (kawikabader) [inline packages/gasket-plugin-https/index.js:78]: @kinetifex @runruh-godaddy would this be more aligned with what you were looking for?

Comment (kinetifex) [reply packages/gasket-plugin-https/index.js:78]: Yea, avoid the IIFE though for readability, just use a reduce:
```js
async function healthCheckRequested() {
  await gasket.exec('healthcheck', HealthCheckError);
}

healthChecks: routes.reduce((acc, cur) => {
  acc[cur] = healthCheckRequested;
  return acc;
}, {})
```

**Review by burgessa23 — APPROVED** [review]

**Comment by kinetifex** [comment]
Looks good and I tested it successfully in the canary-app. Just update the configuration docs with the new defaults then 👍🏻

**Review by kinetifex — APPROVED** [review]

**Pull request merged**